enable/tests/wx/mouse_wheel_test_case.py

```diff
@@ -0,0 +1,79 @@
+
+from unittest import TestCase
+
+from mock import MagicMock
+
+from traits.api import Any
+from traitsui.tests._tools import skip_if_not_wx
+
+from enable.container import Container
+from enable.base_tool import BaseTool
+from enable.window import Window
+
+
+class MouseEventTool(BaseTool):
+
+    event = Any
+
+    def normal_mouse_wheel(self, event):
+        self.event = event
+
+
+@skip_if_not_wx
+class MouseWheelTestCase(TestCase):
+
+    def setUp(self):
+        import wx
+
+        # set up Enable components and tools
+        self.container = Container(postion=[0, 0], bounds=[600, 600])
+        self.tool = MouseEventTool(component=self.container)
+        self.container.tools.append(self.tool)
+
+        # set up wx components and tools
+        self.parent = wx.Frame(None, size=(600, 600))
+        self.window = Window(
+            self.parent,
+            size=(600, 600),
+            component=self.container
+        )
+
+        # Hack: event processing code skips if window not actually shown by
+        # testing for value of _size
+        self.window._size = (600, 600)
+
+    def test_vertical_mouse_wheel(self):
+        import wx
+        from enable.wx.constants import mouse_wheel_axes
+
+        # create and mock a mouse wheel event
+        wx_event = wx.MouseEvent(mouseType=wx.wxEVT_MOUSEWHEEL)
+        wx_event.GetWheelRotation = MagicMock(return_value=200)
+        wx_event.GetWheelAxis = MagicMock(return_value=mouse_wheel_axes[0])
```

Comment:
And presumably we don't test against really old WX versions, what about `wx.MOUSE_WHEEL_VERTICAL` here?